### ISSUE TITLE
STILL spamming console. Could not pass event EntityDeathEvent to MobHunting v6.7.8

### ISSUE THREAD

**Issue by FrozenBeard** [Rocologo/MobHunting#535]
This is about the 4th time ive had to enable/disable MobHunting from the lack of testing thats happening, Every time I try and use it, its just constant spam in the console.

```
[11:35:00] [Server thread/ERROR]: Could not pass event EntityDeathEvent to MobHunting v6.7.8
org.bukkit.event.EventException: null
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.12.2.jar:git-Paper-1617]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:397) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityLiving.die(EntityLiving.java:1148) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityZombie.die(EntityZombie.java:429) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityLiving.damageEntity(EntityLiving.java:983) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityMonster.damageEntity(EntityMonster.java:44) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityZombie.damageEntity(EntityZombie.java:163) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityHuman.attack(EntityHuman.java:1028) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityPlayer.attack(EntityPlayer.java:1397) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1724) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:49) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:6) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_201]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_201]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:850) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
Caused by: java.lang.NoSuchFieldError: ZOMBIE_HEAD
	at one.lindegaard.MobHunting.rewards.CustomItems.getCustomHead(CustomItems.java:369) ~[?:?]
	at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1860) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:35) ~[patched_1.12.2.jar:git-Paper-1617]
	... 23 more

[11:22:06] [Server thread/ERROR]: Could not pass event PlayerFishEvent to MobHunting v6.7.8
java.lang.NoSuchFieldError: SALMON
	at one.lindegaard.MobHunting.FishingManager.Fish(FishingManager.java:110) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor725.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1617]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.EntityFishingHook.j(EntityFishingHook.java:443) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.ItemFishingRod.a(ItemFishingRod.java:19) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.ItemStack.a(ItemStack.java:330) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PlayerInteractManager.a(PlayerInteractManager.java:431) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1064) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:26) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_201]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_201]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:850) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
```

**Comment by Rocologo** [comment]
If you think I am able to test all minecraft versions and all functionality in all combinations, and work on this plugin for free, you are so wrong.

If you think bugs are to annoying you should not update too fast and let others do the testing.

**Comment by Rocologo** [comment]
I will fix this bug to night as well

**Comment by FrozenBeard** [comment]
I have had to update and backdate the plugin for the last 5 releases? I just ended up removing the plugin and waiting for a more stable version that wouldn't spam console, that's the thing, I have been waiting, Why not work on a stable release for all versions and call it stable, then everything else that's added call that alpha is there isn't testing happening on it.

**Comment by Rocologo** [comment]
Im not able to work on two versions (a stable version and and alpha versions) and at the same time add more functionality. The 5 latest versions was mostly for adding support for mc 1.8-mc1.12 and for Reserve.

If you dont need that you should not have updated.

**Comment by FrozenBeard** [comment]
Im using 1.12.2 Paper.

**Comment by Rocologo** [comment]
That explains alot. There is not that many MC 1.12 users left, so testing is limited from my site. 

I have fixed both the ZOMBIE_HEAD bug and the SALMON bug in V6.7.9

I cant promize that all the Material names bugs has been found, but they are easy to fix for me, if you report them.

**Comment by FrozenBeard** [comment]
> There is not that many MC 1.12 users left

Well the actual stats show 15%, 15,000 servers is quite a lot.

![Screenshot 2019-07-27 12 29 43](https://user-images.githubusercontent.com/144046/61993876-50caca00-b06a-11e9-9e6d-d2945e7ddc15.png)

https://bstats.org/global/bukkit

Fantastic, thank you.
I have had to remove the plugin again after updating as users reporting that they are getting MUCH more drops than they should and also some heads are dropping that are the wrong heads, killing a player dropping spider heads etc, I just want a stable version.

**Comment by Rocologo** [comment]
I meant there are not that many MobHunting users on MC 1.12.x if there were 15000 servers it would be very important.

I have no bug reports on killing a player drops spider heads. So if you wanted that to be fixed you should have reported the issue. I almost never play minecraft myself and only on mc 1.14.4, so I need the users to report the problems before I am aware of the problems.

**Comment by Rocologo** [comment]
i think this is fixed in 6.8.0, if not please explain what happens in detils.
